medcat/utils/usage_monitoring.py

```diff
@@ -0,0 +1,101 @@
+import os
+from datetime import datetime
+from typing import List
+import platform
+import logging
+
+from medcat.config import UsageMonitor as UsageMonitorConfig
+
+
+LOGS_ENV = "MEDCAT_LOGS"
```

Comment:
worth renaming to MEDCAT_USAGE_LOGS and MEDCAT_USAGE_LOGS_LOCATION ?
Don't want this to be confused with dev logs(?)

Reply:
Good point!